releasenotes/notes/otlp-tagger-21e4cb2b87eb5e6c.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    OTLP metrics tags are enriched when ```experimental.otlp.metrics.tag_cardinality`` is set to ``orchestrator`` or above.
```

Comment:
Not above, only if set to `orchestrator`, [see here](https://github.com/datadog/datadog-agent/blob/main/pkg/tagger/global.go#L264-L267)